frontend/javascripts/viewer/model/accessors/flycam_accessor.ts

```diff
@@ -297,18 +297,40 @@ function _getFlooredPosition(flycam: Flycam): Vector3 {
   return map3((x) => Math.floor(x), _getPosition(flycam));
 }
 
-function _getRotation(flycam: Flycam): Vector3 {
+// Returns the current rotation of the flycam in radians as an euler xyz tuple.
+// As the order in which the angles are applied is zyx (see flycam_reducer),
+// this order must be followed when this euler angle is applied to 2d computations.
+function _getRotationInRadian(flycam: Flycam, invertZ: boolean = true): Vector3 {
+  // Somehow z rotation is inverted but the others are not.
```

Comment:
Not really sure why z is the one inverted axis, but when I invert the others this break again, especially the bucket picker and the ortho xy plane is no longer consistent to oblique mode plane if this is not applied

Reply:
maybe add a link to the docs.